README.md

```diff
@@ -607,6 +609,27 @@ The exporter will read the password from a secret stored in OCI Vault if you set
 
 > Note that the process must be running under a user that has the OCI CLI installed and configured correctly to access the desired tenancy and region. The OCI Profile used is `DEFAULT`.
 
+### Using Azure Vault
+
+The exporter will read the database username and password from secrets stored in Azure Key Vault if you set these environment variables:
+
+- `AZ_VAULT_ID` should be set to the ID of the Azure Key Vault that you wish to use
```

Comment:
How does Azure support the equivalent of OCI workload identity?

Reply:
see authentication, right below this, it has links to that info